meta/mm_sequence.py

```diff
@@ -175,6 +201,17 @@ def update_rqmt(rqmt, extra):
             else:
                 raise ValueError("Unknown action: %s" % action)
 
+    def get_alignment_score_report(self):
+        """
+        :return: report .txt file path containing the alignment scores in order of the job sequence
+        """
+
+        logs = []
+        for job, log in zip(self.all_jobs, self.all_logs):
+            if isinstance(job, AlignmentJob):
+                logs.append(log)
+        return AMScoresFromAlignmentLogJob(logs).out_report
```

Comment:
Maybe not a good idea to return the `out_report` as then the job instance is not accessible any more. 
(ok, maybe via creator of Path object)

Ideas:
* return the Job and then the caller can get the report out of it.
* add it as a member of the `AlignSplitAccumulateSequence` class
* you can even add it in the constructor of the class and as long as the report is not registered as output the Job will not be created.

Reply:
> add it as a member of the AlignSplitAccumulateSequence class

I will go for this one